standalone/src/main/java/io/pravega/local/InProcPravegaCluster.java

```diff
@@ -47,6 +47,7 @@
 import io.pravega.segmentstore.storage.impl.bookkeeper.ZooKeeperServiceRunner;
 import io.pravega.shared.metrics.MetricsConfig;
 import io.pravega.shared.security.auth.Credentials;
+
```

Comment:
Unnecessary change, please remove.

Reply:
it's added by IntelliJ, I think it's necessary, you can tell this extra blank line separates the imports into 2 parts:
the first part is about the Pravega, the second part is about the java library.